src/data-workspace/data-entry-cell/validation-tooltip.js

```diff
@@ -57,16 +56,16 @@ const TooltipManager = React.forwardRef(function TooltipManager(
 TooltipManager.propTypes = {
     active: PropTypes.bool,
     children: PropTypes.node,
-    invalid: PropTypes.bool,
+    enabled: PropTypes.bool,
     onMouseOut: PropTypes.func,
     onMouseOver: PropTypes.func,
 }
 
-export const ValidationTooltip = ({ children, content, invalid, active }) => {
+export const ValidationTooltip = ({ children, content, enabled, active }) => {
```

Comment:
Renamed `invalid` to `enabled`. After this PR `invalid`-state of the field does not describe the functionality of what this prop actually does. Actually, previously `syncError` was also used to control this prop. The prop actually controls if the tooltip logic is `enabled` - and thus I think that describes the functionality better.